Git-commit: 166ec008033c4e3a1ecf148eb4b10fefff828b71
From: Xie XiuQi <xiexiuqi@huawei.com>
Date: Fri, 26 Feb 2021 20:21:36 +0800
Subject: [PATCH] arm64/mpam: cleanup debuging code
Patch-mainline: Queued in openEuler repo, version 5.10.0-4.5.0
References: openEuler-21.03
Git-repo: https://gitee.com/openeuler/kernel.git

hulk inclusion
category: feature
feature: ARM MPAM support
bugzilla: 48265
CVE: NA

--------------------------------

Cleanup debuging code for sysreg{read,write}, and {read,write}l.
This code is not compiled in current version, and function changed.

There is no change for md5 checking:
md5sum arch/arm64/kernel/mpam*.o

Signed-off-by: Xie XiuQi <xiexiuqi@huawei.com>
Reviewed-by: Hanjun Guo <guohanjun@huawei.com>
Signed-off-by: Yang Yingliang <yangyingliang@huawei.com>
Signed-off-by: Wang ShaoBo <bobo.shaobowang@huawei.com>
Reviewed-by: Cheng Jian <cj.chengjian@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 arch/arm64/include/asm/mpam.h | 35 -----------------------------------
 1 file changed, 35 deletions(-)

diff --git a/arch/arm64/include/asm/mpam.h b/arch/arm64/include/asm/mpam.h
index 102bdc6bcde9..de00c141065f 100644
--- a/arch/arm64/include/asm/mpam.h
+++ b/arch/arm64/include/asm/mpam.h
@@ -244,45 +244,10 @@ static inline int __resctrl_group_show_options(struct seq_file *seq)
 
 void post_resctrl_mount(void);
 
-#ifdef MPAM_SYS_REG_DEBUG
-static inline u64 mpam_read_sysreg_s(u64 reg, char *name)
-{
-	pr_info("cpu %2d (cur: %s(%d)): read_sysreg_s: %s (addr %016llx)\n",
-		smp_processor_id(), current->comm, current->pid, name, reg);
-	return 0;
-}
-#else
 #define mpam_read_sysreg_s(reg, name) read_sysreg_s(reg)
-#endif
-
-#ifdef MPAM_SYS_REG_DEBUG
-static inline u64 mpam_write_sysreg_s(u64 v, u64 reg, char *name)
-{
-	pr_info("cpu %2d (cur %s(%d)): write_sysreg_s: %s (addr %016llx), value %016llx\n",
-		smp_processor_id(), current->comm, current->pid, name, reg, v);
-	return 0;
-}
-#else
 #define mpam_write_sysreg_s(v, r, n) write_sysreg_s(v, r)
-#endif
-
-#ifdef MPAM_SYS_REG_DEBUG
-static inline u32 mpam_readl(const volatile void __iomem *addr)
-{
-	return pr_info("readl: %016llx\n", (u64)addr);
-}
-#else
 #define mpam_readl(addr) readl(addr)
-#endif
-
-#ifdef MPAM_SYS_REG_DEBUG
-static inline u32 mpam_writel(u64 v, const volatile void __iomem *addr)
-{
-	return pr_info("writel: %016llx to %016llx\n", v, (u64)addr);
-}
-#else
 #define mpam_writel(v, addr) writel(v, addr)
-#endif
 
 /**
  * struct msr_param - set a range of MSRs from a domain
-- 
2.31.1

